crates/ra_hir/src/nameres/collector.rs

```diff
@@ -482,10 +482,10 @@ impl<DB> ModCollector<'_, &'_ mut DefCollector<&'_ DB>>
 where
     DB: DefDatabase,
 {
-    fn collect(&mut self, items: &[raw::RawItem]) {
+    fn collect(&mut self, parent_module: Option<&Name>, items: &[raw::RawItem]) {
```

Comment:
Hm, maybe, instead of adding this as parameter to this function, add it as a filed for `ModCollector`? `parent_module: None` reads better than `(None, `

Reply:
Yes.